client/dashboard/app/router.tsx

```diff
@@ -108,7 +138,10 @@ const siteSettingsRoute = createRoute( {
 	getParentRoute: () => siteRoute,
 	path: 'settings',
 	loader: ( { params: { siteSlug } } ) =>
-		queryClient.ensureQueryData( siteSettingsQuery( siteSlug ) ),
+		Promise.all( [
+			queryClient.ensureQueryData( siteQuery( siteSlug ) ),
```

Comment:
Hmm, wouldn't it already ensured in the parent route (`siteRoute`)?

Reply:
It's no longer the parent route though, I moved it to the overview route

Reply:
Sorry, I misread the PR! 🤦

Reply:
It's arguable though. I feel like all site sub routes need access to the basic site object.

Reply:
We can put the new siteQuery (without the other fetches) as a common query. But I'm not sure if every single tab will use it. Can always be changed later